client/app/components/onboardingGuide/styles.scss

```diff
@@ -0,0 +1,131 @@
+.title {
+  font-size: 1.375rem;
+  font-weight: 600;
+  color: #171717;
+  line-height: 1.875rem;
+
+}
+.description {
+  font-size: 1em;
+  font-weight: 400;
+  color: #3c3c3c;
+  line-height: 1.5em;
+  max-width: 550px;
+  width: 90%;
+  margin-bottom: 0.75em;
+}
+
+.sectionSeparator {
+  margin: 40px 0;
+  height: 1px;
+  border-bottom: solid 1px rgba(0,0,0,0.1);
+}
+
+/* GuideStatusPage */
+.stepList {
+  padding-left: 0;
+  margin-bottom: 1rem;
+}
+
+.stepListItem, .stepListItemDone {
+  list-style: none;
+}
+
+.stepListCheckbox {
+  background-image: url('../../assets/images/iconSquare@2x.png');
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position-y: 1px;
+  display: inline-block;
+  width: 1rem;
+  height: 1rem;
+  margin-right: 0.5rem;
+}
+.stepListItemDone .stepListCheckbox {
+  background-image: url('../../assets/images/iconSquareChecked@2x.png');
+  @extend .stepListCheckbox;
+}
+
+.stepListLink {
+  font-size: 1.125rem;
+  font-weight: 600;
+  color: #59b3a2;
+  line-height: 2.25rem;
+
+  &:hover {
+    color: #4d998b;
+  }
+}
+.stepListItemDone .stepListLink {
+  color: #646464;
+  text-decoration: line-through;
+}
+
+.nextButton {
+  display: inline-block;
+  margin: 2rem 0;
+  border: 0;
+  border-radius: 5px;
+  font-size: 1rem;
+  font-weight: 600;
+  text-decoration: none;
+  color: white;
+  background-color: #59b3a2;
+  padding: 0.626rem 1.875rem;
+
+  &:hover {
+    background-color: #4d998b;
+    color: white;
+  }
+}
+
+/* Common elements for step related pages */
+.backLink {
+  font-size: 0.875rem;
+  font-weight: 600;
+  color: #59b3a2;
+  &:hover {
+    color: #4d998b;
+  }
+}
+
+.infoTextContainer {
+  position: relative;
+  color: gray;
+  font-size: 0.8125em;
+  line-height: 1.38462em;
+  margin-top: 0.375em;
+  max-width: 550px;
+  width: 90%;
+
+  .infoTextIcon {
+    position: absolute;
+    width: 0.75em;
+    margin-top: 1px;
+  }
+
+  .infoTextContent {
+    padding-left: 1.5em;
+  }
+}
+
+.sloganImageContainer {
+  margin: 2rem 0;
+  display: block;
+  width: 100%;
+  height: auto;
+  position: relative;
+  overflow: hidden;
+  padding: 46.72% 0 0 0; /* 46.72% = 100 / (w / h) = 100 / (732 / 342) */
```

Comment:
Apparently, there is. e.g. https://www.npmjs.com/package/postcss-aspect-ratio
But since we have not yet added PostCSS support, I will do it manually. 
(But this is good to remember when we really start developing new design with PostCSS.)